Source/Core/DiscIO/VolumeDirectory.cpp

```diff
@@ -460,18 +461,18 @@ void CVolumeDirectory::WriteDirectory(const File::FSTEntry& parent_entry, u32* f
     // For some reason, sorting by lowest ASCII value first prevents many games from
     // fully booting. We make the comparison case insensitive to solve the problem.
     // (Highest ASCII value first seems to work regardless of case sensitivity.)
```

Comment:
Please remove this comment. Case-sensitive sorting seems to be working fine now thanks to your off-by-one fix. (But let's keep using case-insensitive sorting.)